md/optimize-user-tasklist.md

```diff
@@ -151,14 +164,15 @@ When reusing these features for your own projects, make sure that the result of
 ## View it in the task list
 
 1. Save the diagram
-2. Run the pool
+2. Run the pool. The default loggued user is Walter Bates
 3. Fill out the default instantiation form with two dates
 4. Click on **Start**  
 
 Once you are sent to the task list, you can see that the task name is contextualized.  
 It is computed once, when the task becomes ready. 
 
 **Note:** The following set of instructions only applies to the task list in Bonita BPM 7.3.0 and above. In earlier versions, the **Description** field is displayed by default.  
```

Comment:
I thought it was too big an emphasis. Those wells are real attention focus traps. We should use them with care.